ocs_ci/framework/logger_factory.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from ocs_ci.framework import config
+
+current_factory = logging.getLogRecordFactory()
+
+
+def record_factory(*args, **kwargs):
+    """
+    Record factory setup function
+    Return:
+        logging.record: Reference obj to the record of logger
+    """
+    record = current_factory(*args, **kwargs)
+    record.clusterctx = f"C[{config.cur_index}]"
```

Comment:
I feel ```record.clusterctx = f"[{config.ENV_DATA['cluster_name'}]"``` would make logs more readable. WDYT?

Reply:
I agree it could be more readable, WDYT? @petr-balogh @clacroix12